RenderStateNotationParser/include/RenderStateNotationParserImpl.hpp

```diff
@@ -33,14 +35,18 @@
 namespace Diligent
 {
 
+using IncludeHandlerRenderStateNotationParser = std::unordered_set<std::string>;
```

Comment:
This should be a private type of `IRenderStateNotationParserImpl` - it is not used outside of parser implementation and should not be in the common namespace